vite.config.ts

```diff
@@ -22,6 +22,7 @@ export default defineConfig({
     target: ['es2019'],
     minify: true,
     reportCompressedSize: true,
+    commonjsOptions: { transformMixedEsModules: true },
```

Comment:
yikes, maybe another reason to check the other library out.  I think this is kind of an indicator that the library is a bit screwy.